src/uri.ts

```diff
@@ -34,27 +31,45 @@ export function resolveURI(uri: string): ResolvedURI {
  */
 export function codecovParamsForRepositoryCommit(
     uri: Pick<ResolvedURI, 'repo' | 'rev'>
-): Pick<CodecovGetCommitCoverageArgs, 'service' | 'owner' | 'repo' | 'sha'> {
-    // TODO: Support services (code hosts) other than GitHub.com, such as GitHub Enterprise, GitLab, etc.
-    if (uri.repo.startsWith('github.com/')) {
+): Pick<CodecovGetCommitCoverageArgs, 'baseURL' | 'service' | 'owner' | 'repo' | 'sha'> {
+    try {
+        const endpoints: Endpoint[] | undefined = sourcegraph.configuration.get<Settings>().get('codecov.endpoints')
+        const baseURL: string = endpoints && endpoints[0] && endpoints[0].url || ''
+
+        const knownHosts: any[] = [
```

Comment:
Is there a reason you made this `any[]` instead of letting the type be inferred?